dags/public-health/Code75.py

```diff
@@ -0,0 +1,64 @@
+import arcgis
+import sqlalchemy
+
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.postgres_hook import PostgresHook
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2020, 1, 30),
+    "email": [
+        "ian.rose@lacity.org",
+        "hunter.owens@lacity.org",
+        "brendan.bailey@lacity.org",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(days=1),
+}
+
+dag = DAG("code-75", default_args=default_args, schedule_interval="@daily")
+
+def update_code_75(**kwargs):
+    #Connecting to ArcGIS
+    arcconnection = BaseHook.get_connection("arcgis")
+    gis = arcgis.GIS("http://lahub.maps.arcgis.com/", arcconnection.login, arcconnection.password)
+    
+    #Getting Code 75 data
+    gis_item = gis.content.get(kwargs.get("arcfeatureid"))
+    sdf = gis_item.layers[0].query().sdf
+    sdf.drop("SHAPE", axis =1, inplace = True)
```

Comment:
An aside: how do you like working with `SpatialDataFrame`s? I haven't really spent much time with them. Do they compare nicely with `GeoDataFrame`s?

Reply:
This was my first experience using a spatial dataframe. I just used it because it's already part of the ArcGIS library.